charts/telegraf/templates/servicemonitor.yaml

```
@@ -0,0 +1,22 @@
+{{- if .Values.serviceMonitor.create }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "telegraf.serviceMonitorName" . }}
+  labels:
+    {{- include "telegraf.labels" . | nindent 4 }}
```

I feel like it would be handy to allow users specify additional custom labels as well, for more relaxed monitor referencing from Prometheus, what do you think? I noticed Grafana template has such option.

```suggestion
    {{- include "telegraf.labels" . | nindent 4 }}
    {{- if .Values.serviceMonitor.labels }}
      {{- toYaml .Values.serviceMonitor.labels | nindent 4 }}
    {{- end }}    
```

---

done, let me know when its ok